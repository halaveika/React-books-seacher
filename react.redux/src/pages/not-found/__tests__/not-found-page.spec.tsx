import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import NotFoundPage from '../not-found-page';
Enzyme.configure({ adapter: new Adapter() });

describe('test NotFoundPage component', () => {
  it('should render NotFoundPage component', async () => {
    const wrapper = shallow<NotFoundPage>(<NotFoundPage title="test" />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
