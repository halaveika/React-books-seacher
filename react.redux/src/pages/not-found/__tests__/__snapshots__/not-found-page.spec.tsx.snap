// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test NotFoundPage component should render NotFoundPage component 1`] = `
<Fragment>
  <Header
    title="test"
  />
  <div
    className="container"
  >
    <div
      className="content"
    >
      <img
        alt="page_not_found"
        className="not-found-image"
        src="/assets/images/error-404.png"
      />
    </div>
  </div>
</Fragment>
`;
